### PR TITLE
[zh]Resync Reference files[11]

### PR DESCRIPTION
**Pull request by CaoDonghui123**
part of #27673

### CODE REVIEW

File: content/zh/docs/reference/_index.md [hunk 1]
```diff
@@ -48,8 +59,10 @@ client libraries:
 - [Kubernetes Python client library](https://github.com/kubernetes-client/python)
 - [Kubernetes Java client library](https://github.com/kubernetes-client/java)
 - [Kubernetes JavaScript client library](https://github.com/kubernetes-client/javascript)
+- [Kubernetes Dotnet client library](https://github.com/kubernetes-client/csharp)
```
Comment (HongjiangHuang) [inline content/zh/docs/reference/_index.md:62]: Csharp is not Dotnet.

https://docs.microsoft.com/en-us/dotnet/core/introduction#programming-languages

Comment (CaoDonghui123) [reply content/zh/docs/reference/_index.md:62]: >The generated code works on all platforms supported by .NET or .NET Core.

https://github.com/kubernetes-client/csharp#generating-the-client-code

File: content/zh/docs/reference/labels-annotations-taints.md [hunk 2]
```diff
@@ -316,6 +365,16 @@ Starting in v1.18, this annotation is deprecated in favor of `spec.ingressClassN
...
+## storageclass.kubernetes.io/is-default-class
+
+示例：`storageclass.kubernetes.io/is-default-class=true`
+
+用于：StorageClass
+
+<!--
+When a single StorageClass resource has this annotation set to `"true"`, new Physical Volume Claim resource without a class specified will be assigned this default class.
+-->
+当单个的 StorageClass 资源将这个注解设置为 `"true"` 时，新的物理卷没有声明指定类别的资源，将被分配为此默认类别。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints.md:377]:
```suggestion
当单个的 StorageClass 资源将这个注解设置为 `"true"` 时，新的持久卷申领（PVC）
资源若未指定类别，将被设定为此默认类别。
```

File: content/zh/docs/reference/labels-annotations-taints.md [hunk 3]
```diff
@@ -327,14 +386,47 @@ Starting in v1.18, this annotation is deprecated in favor of `spec.ingressClassN
...
+
+示例：`batch.kubernetes.io/job-completion-index: "3"`
+
+用于：Pod
+
+<!--
+The Job controller in the kube-controller-manager sets this annotation for Pods
+created with Indexed [completion mode](/docs/concepts/workloads/controllers/job/#completion-mode).
+-->
+kube-controller-manager 中的 Job 控制器给创建使用索引[完成模式](/zh/docs/concepts/workloads/controllers/job/#completion-mode)
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints.md:405]:
```suggestion
kube-controller-manager 中的 Job 控制器给创建使用索引
[完成模式](/zh/docs/concepts/workloads/controllers/job/#completion-mode)
```

File: content/zh/docs/reference/labels-annotations-taints.md [hunk 4]
```diff
@@ -327,14 +386,47 @@ Starting in v1.18, this annotation is deprecated in favor of `spec.ingressClassN
...
+的 Pod 设置此注解。
+
+## kubectl.kubernetes.io/default-container
+
+示例：`kubectl.kubernetes.io/default-container: "front-end-app"`
+
+<!--
+The value of the annotation is the container name that is default for this Pod. For example, `kubectl logs` or `kubectl exec` without `-c` or `--container` flag will use this default container.
+-->
+注解的值是此 Pod 的默认容器名称。例如，`kubectl logs` 或 `kubectl exec` 没有 `-c` 或 `--container` 参数时，将使用这个默认的容器。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints.md:415]:
```suggestion
注解的值是此 Pod 的默认容器名称。
例如，`kubectl logs` 或 `kubectl exec` 没有 `-c` 或 `--container` 参数时，将使用这个默认的容器。
```

File: content/zh/docs/reference/labels-annotations-taints.md [hunk 5]
```diff
@@ -327,14 +386,47 @@ Starting in v1.18, this annotation is deprecated in favor of `spec.ingressClassN
...
+示例：`endpoints.kubernetes.io/over-capacity:warning`
+
+用于：Endpoints
+
+<!-- 
+In Kubernetes clusters v1.21 (or later), the Endpoints controller adds this annotation to an Endpoints resource if it has more than 1000 endpoints. The annotation indicates that the Endpoints resource is over capacity.
+
+**The taints listed below are always used on Nodes**
+-->
+在 Kubernetes 集群 v1.21（或更高版本）中，如果 Endpoint 超过 1000 个，Endpoint 控制器就会向其添加这个注解。该注解表示 Endpoint 资源已超过容量。
```
Comment (tengqm) [inline content/zh/docs/reference/labels-annotations-taints.md:428]:
```suggestion
在 Kubernetes 集群 v1.21（或更高版本）中，如果 Endpoint 超过 1000 个，Endpoint 控制器
就会向其添加这个注解。该注解表示 Endpoint 资源已超过容量。
```

File: content/zh/docs/reference/_index.md (hunk 1 above)
Comment (HongjiangHuang) [reply content/zh/docs/reference/_index.md:62]: https://github.com/kubernetes/website/pull/28250

😅The java client compiles code for all JVM platform languages, 
but it's still called the java client, not the JVM client.

**Pull request merged**